_pages/sponsorship.html

```diff
@@ -9,7 +9,7 @@
     <h1>Sponsor our team!</h1>
     <p>Without our generous sponsors, we would not have the opportunity to get the most out of this wonderful program. The funds and other assistance provided by people like you helps our team purchase the tools and parts we need to build our robots, enter competitions, and, of course, order pizza during build season.</p>
     <div class="button-container">
-        <a class="button" href="mailto:robotics1418ATSYMBOLgmailPERIODcom?subject=Interested in sponsorship of Team 1418">Contact us about Sponsorship</a>
+        <a class="button" href="//INSERT DOWNLOAD LINK FOR SPONSOR INFO PACKET HERE//">Learn more about sponsoring us</a>
```

Comment:
Not merging this yet for obvious reasons...

Reply:
Yea, check Discord I sent the sponsorship packet to you to add to assets